src/components/AccessibilityPolicyForm/AccessibilityPolicyForm.test.jsx

```diff
@@ -28,7 +27,7 @@ const validationMessages = {
   message: 'Enter a message.',
 };
 
-const dangerIconDescription = 'Error: ';
```

Comment:
I noticed that it seems like a lot of trailing punctuation has been removed from the tests. Why is that? Is there something about internationalizing punctuation?

Reply:
This specific one I can answer -- This text was for the screenreader only text, which doesn't need punctuation (doesn't get read anyway) also trailing punctuation doesn't always work the same way in each language / might not make sense at all with how that language uses punctuation at all. This one specifically was mostly for general clean up purposes, but we can revisit other instances as well.

Reply:
Ah, I see. Thanks for the clarification!